src/_scss/pages/search/results/table/_tableTypes.scss

```diff
@@ -29,5 +29,53 @@
             border-left: 1px solid $color-gray-light;
             color: $color-base;
         }
+        &.coming-soon {
+            background: repeating-linear-gradient(
+                45deg,
+                rgba($color-gray-lightest, .2),
+                rgba($color-gray-lightest, .2) rem(2px),
+                rgba($color-gray-lighter, .3) rem(2px),
+                rgba($color-gray-lighter, .3) rem(4px)
+            );
+            color: $color-gray-light;
+            .coming-soon-container {
+                width: 100%;
+                text-align: center;
+                height: 10px;
+            }
+            .coming-soon-icon {
+                bottom: rem(0px);
+                display: inline;
+                left: rem(0px);
+                svg {
+                    fill: $color-gray-lighter;
+                    height: rem(10px);
+                    width: rem(10px);
+                }
+            }
+            .coming-soon-label {
+                bottom: rem(2px);
+                color: $color-gray-lighter;
+                display: inline;
+                font-weight: bold;
+                font-size: rem(10px);
+                height: 0;
+                left: rem(3px);
+                position: relative;
+            }
+            &:hover {
+                background: repeating-linear-gradient(
+                    45deg,
+                    rgba($color-gray-lightest, .2),
+                    rgba($color-gray-lightest, .2) rem(2px),
+                    rgba($color-gray-lighter, .3) rem(2px),
+                    rgba($color-gray-lighter, .3) rem(4px)
+                );
+                cursor: default;
+            }
+            &:hover * {
```

Comment:
Same as above - also I think in this case, the wildcard hover is just duplicating the above hover class.